workspaces/services/guacamole-azure-win10vm/terraform/main.tf

```diff
@@ -36,3 +36,8 @@ data "azurerm_subnet" "services" {
   virtual_network_name = data.azurerm_virtual_network.ws.name
   resource_group_name  = data.azurerm_resource_group.ws.name
 }
+
+data "azurerm_key_vault" "kv" {
+  name                = "kv-${var.tre_id}-${var.workspace_id}"
```

Comment:
Looks like this isn't the same name you use when creating the resource.

Reply:
Fixed